app/views/layouts/application.html.haml

```diff
@@ -16,5 +16,8 @@
       .box
         %span.alert= alert
     = yield
-    .logo.col-center-block
-      = link_to image_tag('logo.png'), 'http://www.craftacademy.se', target: '_blank'
+    .logo.center
+      %p
+        A
+        = link_to 'CRAFT ACADEMY', 'http://www.craftacademy.se', target: '_blank'
+        PRODUCTION
```

Comment:
I think it's better to write in regular case in the semantic HTML and then do a `text-transform: uppercase;` with CSS